build.gradle

```
@@ -792,15 +616,20 @@ task mapNamedJar(type: MapJarTask, dependsOn: ["mergeV2", "unpickIntermediaryJar
 	]
 }
 
+task syncDependencies(type: Sync) {
+	from configurations.minecraftLibraries
+	into '.gradle/temp/mclibs'
```

```suggestion
	into 'build/tmp/mclibs'
```
This is where many other temp files go